app/components/AmountsPanel/AmountsPanel.scss

```diff
@@ -1,17 +1,62 @@
 @import '../../styles/variables';
 
-.amountsPanel {
+.zeroAmount {
   display: flex;
-  flex-wrap: wrap;
-  flex: 1;
   align-items: center;
   justify-content: center;
+  background-color: #f2f2f2;
+
+  svg {
+    height: 36px;
+    width: 36px;
+    margin-right: 15px;
+  }
+}
+
+h1 {
+  margin-top: 0;
+  font-family: Gotham-Light;
+  font-weight: 100;
+  font-size: 30px;
+  color: #8d98ae;
+  line-height: 32px;
+  margin-bottom: 0;
+}
+
+.container,
+.zeroAmount {
+  margin-bottom: 20px;
+}
+
+.container {
```

Comment:
Should have a background color of `#f2f2f2` so that when expanding the window the layout doesnt look "off"